projects/storefrontstyles/scss/_components.scss

```diff
@@ -49,7 +51,7 @@ $selectors: mergeAll(
 // add body specific selectors
 body {
   @each $selector in $selectors {
-    @if (index($skipComponentStyles, $selector) == null) {
```

Comment:
same as above

Reply:
As above

Reply:
same as above

Reply:
This is a specific commit and has been resolved in a later commit